src/player.js

```diff
@@ -167,8 +167,12 @@ class Player {
         return this.findCountFct((card) => { return card.name == cardToFind.name; });
     }
 
+    findCountBaseCard(cardToFind) {
+        return this.findCountFct((card) => { return card.name == cardToFind.name || card.baseCardName === cardToFind.name; });
+    }
+
     findCountHusbandMatches() {
-        return this.getWifeCount4AdditionalPoints() + this.findCountCard(basecards['mayberrymatriarch']);
+        return this.getWifeCount4AdditionalPoints();
```

Comment:
@tadeaustria Shouldn't this be left unchanged?

Reply:
Other than that I think everything is ok

Reply:
Yes, thanks you are right. Without it, the Mayberry didn't take up a space if placed alone.